src/styles/index.scss

```diff
@@ -1,3 +1,37 @@
+@import './utils/variables';
+@import './blocks/header';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  box-sizing: border-box;
+  display: block;
+}
```

Comment:
The body styles should be moved to a separate file in the utils folder. This will make your code more organized and easier to maintain.